freertos-rust-examples/examples/stm32-cortex-m4-blackpill/main.rs

```diff
@@ -59,6 +59,10 @@ impl<D1: OutputPin> MyDevice<D1>
 
 #[entry]
 fn main() -> ! {
+    //Demo availability of cpu_clock_hz API
+    #[cfg(feature = "cpu_clock")]
+    cpu_clock_hz();
+
```

Comment:
This is good as a proof for this change, but for merging in should be removed as it doesn't add anything useful for this example.